util/EIP-2124/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+//! EIP-2124 implementation based on <https://eips.ethereum.org/EIPS/eip-2124>
+
+use crc::crc32;
+use ethereum_types::H256;
+use rlp::RlpStream;
+
+pub type BlockNumber = u64;
+
+#[derive(Clone, Copy, Debug)]
+pub struct ForkHash(pub u32);
+
+impl rlp::Encodable for ForkHash {
+	fn rlp_append(&self, s: &mut RlpStream) {
+        s.append_iter(self.0.to_be_bytes().iter().copied());
+    }
```

Comment:
inconsistent indentation
(we use tabs everywhere else)

Reply:
fixed